gobblin-modules/gobblin-kafka-09/src/main/java/org/apache/gobblin/kafka/writer/Kafka09DataWriter.java

```diff
@@ -126,4 +138,33 @@ public void flush()
       throws IOException {
     this.producer.flush();
   }
+  
+  private void provisionTopic(String topicName,Config config) {
+		String zooKeeperPropKey = KafkaWriterConfigurationKeys.KAFKA_TOPIC_CONFIG+KafkaWriterConfigurationKeys.CLUSTER_ZOOKEEPER;
+		if(!config.hasPath(zooKeeperPropKey)){
+		    LOG.debug("Topic "+topicName+" is configured without the partition and replication");
+			return;
+		}
+	    String zookeeperConnect = config.getString(zooKeeperPropKey);
+	    int sessionTimeoutMs = 10 * 1000;
```

Comment:
Please make these configurable with default values. At least these should be static constants.